### PR TITLE
LPS-199429: Add context title to back button when editing a Web Content

### PR DESCRIPTION
**Pull request by DiegoHu97**
[Link to ticket](https://liferay.atlassian.net/browse/LPS-199429)

## Steps 

**NOTE: Alll the pages that take us to edit a Web Content, the title it set in JournalEditArticleDisplayContext.java**
**Case 1 AssetHelperImpl.java: Add WC from Page Collection**

- Create a Manual Collection with item type Web Content Article and Item Subtype Basic Web Content.
- Go to Site Builder >Pages
- Create a Collection Page named “TestCollection” with the Manual Collection created
- In edit mode of the page > Click on the + icon next to the name of the page.
- The edit mode of a Web Content is automatically open.
- Hover over the back button
Screenshot 2023-11-23 at 15 37 27

**Case 2 JournalManagementToolbarDisplayContext.java: Create WC from Web Content**

- Go to Content & Data > Web Content > click on plus button > see back button title
Screenshot 2023-11-23 at 15 41 03

**Case 3 JournalArticleVerticalCard.java: Edit WC in Cards view**

- Create a WC > Go to Web Contents view
- Next to plus icon > click on view mode > select Cards
- Click on the name of the WC > see back button 
Screenshot 2023-11-23 at 15 43 17

**Case 4 view_entries.jsp: edit WC in list and table view**

- Create a WC > Go to Web Contents view
- Next to plus icon > click on view mode > select list and then table
- Click on the name of the WC > see back button 
Screenshot 2023-11-23 at 15 44 52

**Case 5 JournalArticleActionDropdownItemsProvider.java - _getEditArticleActionUnsafeConsumer(): edit WC from dropdown**

- Create a WC > Go to Web Contents view
- Click on WC ellipsis > edit > see back button 
Screenshot 2023-11-23 at 15 46 32

**Case 6: Importing translation for WC**
**Case 6.1: JournalArticleActionDropdownItemsProvider.java - _getImportTranslationActionUnsafeConsumer()**

- Create a WC > Go to Web Contents view
- Click on WC ellipsis > Import Translation > see back button 
Screenshot 2023-11-23 at 15 48 51

**Case 6.2: ImportTranslationPortletConfigurationIcon.java** 

- Create a WC > Go to Web Contents view
- Upper in Control Menu > click on ellipsis > Import Tranlations
Screenshot 2023-11-23 at 15 49 55

**Case 7 JournalArticleActionDropdownItemsProvider.java - _getExportForTranslationActionUnsafeConsumer(): export WC for translation**

- Create a WC > Go to Web Contents view
- Click on WC ellipsis > Export for Translation > see back button 
Screenshot 2023-11-23 at 15 51 29

**Case 8 JournalArticleActionDropdownItemsProvider.java - _getTranslateActionUnsafeConsumer(): translate WC**

- Create a WC > Go to Web Contents view
- Click on WC ellipsis > Translate > see back button 
Screenshot 2023-11-23 at 15 54 38

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/translation/translation-web/src/main/resources/META-INF/resources/export_translation.jsp [hunk 1]
```diff
@@ -8,12 +8,13 @@
 <%@ include file="/init.jsp" %>
 
 <%
-portletDisplay.setShowBackIcon(true);
-
```
Comment (BarbaraCabrera) [inline modules/apps/translation/translation-web/src/main/resources/META-INF/resources/export_translation.jsp:12]: Maybe we can add here the setURLBackTitle() to follow the alphabetical order

**Comment by BarbaraCabrera** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:test:stable

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**